test/CMakeLists.txt

```diff
@@ -236,6 +236,15 @@ add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/nh_ar_tiles/2/000/765/07
 add_custom_target(nh_ar_tiles DEPENDS ${CMAKE_BINARY_DIR}/test/data/nh_ar_tiles/2/000/765/074.gph)
 set_target_properties(nh_ar_tiles PROPERTIES FOLDER "Tests")
 
+add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/universitatstunnel/2/000/812/907.gph
+  COMMAND ${CMAKE_COMMAND} -E make_directory test/data/universitatstunnel/
+  COMMAND ${CMAKE_BINARY_DIR}/valhalla_build_tiles
+      --inline-config '{"mjolnir":{"id_table_size":1000,"tile_dir":"test/data/universitatstunnel","timezone":"test/data/tz.sqlite","hierarchy":true,"shortcuts":true,"concurrency":1,"logging":{"type":""}}}'
+      ${VALHALLA_SOURCE_DIR}/test/data/universitatstunnel.osm.pbf
+  COMMENT "Building Universitatstunnel Tiles..."
+  WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+add_custom_target(de_universitatstunnel_tiles DEPENDS ${CMAKE_BINARY_DIR}/test/data/universitatstunnel/2/000/812/907.gph)
+set_target_properties(de_universitatstunnel_tiles PROPERTIES FOLDER "Tests")
```

Comment:
unfortunately this wasnt the right way to test this, it should have been done with a gurka test, we'll want to port this. @gknisely do you or anyone else have spare cycles?